CHANGELOG

```diff
@@ -35,6 +35,8 @@ Version 4.3.1 (development)
 
 - Fixed several MinGW build issues on Windows.
 
+- Remove the 'u' flag in the ar command, to update all files in the archive.
```

Comment:
Maybe explain why this change is made here?

Reply:
`done`, let me know if that is what you had in mind.

Reply:
Looks good. Might be useful to mention that this issue comes up when compiling with nvcc. What do you think?

Reply:
isn't more related to the `ar` version on `linux` vs the `macos`, but depending on the compiler (`nvcc` is one)?

Reply:
Ah, yeah, you're right. Ok, it's good as it is now.